client/landing/subscriptions/tab-views/SiteRow.tsx

```diff
@@ -0,0 +1,21 @@
+import { SiteType } from './site-types';
```

Comment:
I know in the past I got you to use the Pascal case for file names, but we should actually be using kebab_case instead, as the rest of the calypso. 🙈

Reply:
Done 👍